drivers/espi/espi_npcx.c

```diff
@@ -552,32 +592,57 @@ static bool espi_npcx_channel_ready(const struct device *dev,
 static int espi_npcx_send_vwire(const struct device *dev,
 			enum espi_vwire_signal signal, uint8_t level)
 {
+	uint8_t reg_idx, bitmask, sig_idx, val = 0, vw_tbl_size;
 	struct espi_reg *const inst = HAL_INSTANCE(dev);
-	uint8_t reg_idx, bitmask, sig_idx, val = 0;
+	const struct npcx_vw_out_config *vw_tbl;
+	uint32_t reg_val;
+	char *reg_name;
+
+	if (signal >= ESPI_VWIRE_SIGNAL_SLV_GPIO_0) {
```

Comment:
Should there be an extra first case for signal `>= ESPI_VWIRE_SIGNAL_COUNT`? I understand it would fail the check below as well, but thinking it may be nice to have a more specific error message here since you have that constant now.